.github/workflows/build_docs.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           pip config --site set global.progress_bar off
           python -m pip install --upgrade pip
           python -m pip install -e .
-          python -m pip install -r dev-requirements.txt
+          python -m pip install -e .[dev]
```

Comment:
I think you can just do `python -m pip install .[dev]` here without the `python-m pip install -e .` command. Also, I don't think there is a need to install this in editable mode.

Reply:
updated